apps/web/src/app/[locale]/(profile)/[username]/_components/profile-skeleton.tsx

```diff
@@ -0,0 +1,40 @@
+import { Skeleton } from '@repo/ui/components/skeleton';
+
+export const ProfileSkeleton = () => {
```

Comment:
suggestion: Could we stick to arrow functions or regular functions?

Reply:
lets use function declaration for component definitions